docker-compose.yaml

```diff
@@ -23,7 +23,7 @@ services:
     restart: on-failure
 
   kakarot-deployer:
-    image: ghcr.io/kkrt-labs/kakarot/deployer:v0.6.2
+    image: ghcr.io/kkrt-labs/kakarot/deployer:latest
```

Comment:
We can't have latest can we? Otherwise it will pull the latest from Kakarot which won't always match on rpc

Reply:
it's the latest 'release' no?

Reply:
but yes, fair
```suggestion
    image: ghcr.io/kkrt-labs/kakarot/deployer:v0.7.2
```